### ISSUE TITLE
link checkboxes to text again

### ISSUE THREAD

**Issue by mycroftw** [mycroftw/conv-cards#74]
Almost all of the checkboxes and their meanings are both manually set.  This was 100% necessary back in the \pfix{} days because otherwise there would be some boxes "below the line" and some "on the line" and it looks stupid.

My latest feeling was that (node.east)ing the checkboxes led to poor formatting - the boxes were sometimes a long way away from the text, sometimes really close, and looked ugly.

But with the \pfix{} being unnecessary and the text being aligned properly by default, maybe this isn't necessary any more.

Also, given the way tikz is intended to work, it's probably best to "node" the checkboxes and fixed-orient the text as a "label".  So we should try this.

something like:
```latex
\checkbox (clcon) at (100, 191)
\node at (onec5.west) [t] \tRedreg{Conv}
```

**Comment by mycroftw** [comment]
Note: changing the anchor to the checkbox will require changing the checkbox commands to allow the second argument as a label.  If the initial argument can be used as text as well as \boolean{#1}, then the second argument isn't needed - but there will still be a change to the checkbox command.  It won't affect cards, just the style file itself, so this isn't a breaking change, and maybe should be done straight up for value purposes.

**Comment by mycroftw** [comment]
Actually it doesn't. We can clearly do `\checkbox{1ntw} at (100, 100) (1ntwbox)` already.  But maybe I should anyway if possible, to get automatic labels?

**Comment by mycroftw** [comment]
See also https://tex.stackexchange.com/questions/58878/tikz-set-node-label-position-more-precisely for "setting the label position on a single node more precisely".

**Comment by mycroftw** [comment]
Issue resolved by #79
